tests/setup.js

```diff
@@ -0,0 +1,26 @@
+'use strict';
+
+const { getHttpOperationsFromSpec }  = require('@stoplight/prism-cli/dist/operations');
+const { createClientFromOperations } = require('@stoplight/prism-http/dist/client');
+const { URL }                        = require('url');
+
+const lobUri = 'https://api.lob.com/v1';
+const clientOptions = {
+  mock: false,
+  validateRequest: true,
+  validateResponse: true,
+  checkSecurity: false,
+  errors: true,
+  baseurl: lobUri,
+  upstream: new URL(lobUri),
+};
+
+const specFile = './Lob-API-public.yml';
+
+module.exports.setup = async () => {
+  const operations = await getHttpOperationsFromSpec(specFile);
+  return createClientFromOperations(operations, clientOptions);
+}
+
+const testSecret = 'BASIC dGVzdF8wZGM4ZDUxZTBhY2ZmY2IxODgwZTBmMTljNzliMmY1YjBjYzo='
```

Comment:
I think there might be a linter command that looks for keys like this.

Reply:
nvr mind. looks like this particular endpoint is meant to be public for now. I'll approve.

Reply:
It's the key from the developer docs site, so not adding any additional badness. It's also temporary until we do the ticket that pulls the token from the environment and does the base64 encoding inline. Which is all in the README. :smile:

Reply:
Nonetheless, I super appreciate your careful reading and concern for security!